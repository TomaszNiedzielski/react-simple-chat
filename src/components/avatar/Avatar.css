.chat-message__avatar {
    height: 30px;
    width: 30px;
    margin-bottom: 2px;
}
.chat-message__avatar img {
    height: 30px;
    width: 30px;
    border-radius: 50%;
    object-fit: cover;
}