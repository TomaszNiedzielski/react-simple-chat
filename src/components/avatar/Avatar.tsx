import React from 'react';

interface Props {
    source: string;
}

const Avatar: React.FC<Props> = ({ source }) => {
    return (
        <div className="chat-message__avatar">
            <img src={source} alt="avatar" />
        </div>
    );
}
export default Avatar;