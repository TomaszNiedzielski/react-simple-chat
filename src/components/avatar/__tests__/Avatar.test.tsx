import renderer from 'react-test-renderer';
import Avatar from '../Avatar';

it('renders bubble avatar', () => {
    const component = renderer.create(
        <Avatar source="" />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});