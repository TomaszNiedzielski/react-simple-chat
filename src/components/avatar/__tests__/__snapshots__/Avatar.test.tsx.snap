// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bubble avatar 1`] = `
<div
  className="chat-message__avatar"
>
  <img
    alt="avatar"
    src=""
  />
</div>
`;
